src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,49 @@
+package wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+@Entity
+@Data
+@NoArgsConstructor
+@ToString
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE)
+    private long id;
+
+    @Column(nullable = false)
```

Comment:
We recommend using javax.validation.constraints.NotNull because it checks that the value is not null before getting to the persistance layer